airflow/ui/src/queries/useAddVariable.ts

```diff
@@ -0,0 +1,67 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { useQueryClient } from "@tanstack/react-query";
+import { useState } from "react";
+
+import {
+  useVariableServiceGetVariablesKey,
+  useVariableServicePostVariable,
+} from "openapi/queries";
+import { toaster } from "src/components/ui";
+import type { AddVariableBody } from "src/pages/Variables/AddVariableForm";
+
+export const useAddVariable = (onClose: () => void) => {
+  const queryClient = useQueryClient();
+  const [error, setError] = useState<unknown>(undefined);
+
+  const onSuccess = async () => {
+    await queryClient.invalidateQueries({
+      queryKey: [useVariableServiceGetVariablesKey],
+    });
+
+    toaster.create({
```

Comment:
Actually I am to spot the error but no taoster message was displayed when adding a variable.

Reply:
fixed.